samples/ra-vpn/redirector-lb/ca-certificate.yaml

```diff
@@ -1,3 +1,12 @@
+# This certificate is provided by you. It is a PKCS12 formatted cert
+# generated on the domains used in your Route53Ingress config. 
+# For VPN Redirector tolology, this should be a wildcard cert which 
```

Comment:
`# For VPN Redirector topology, this should be a wildcard cert which `

Reply:
fixed, thanks